packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -160,8 +156,11 @@ contract PerpetualPositionManager is FeePayer, FundingRateApplier, Administratee
         require(_getIdentifierWhitelist().isIdentifierSupported(_priceIdentifier), "Unsupported price identifier");
 
         withdrawalLiveness = _withdrawalLiveness;
-        TokenFactory tf = TokenFactory(_tokenFactoryAddress);
-        tokenCurrency = tf.createToken(_syntheticName, _syntheticSymbol, 18);
+        tokenCurrency = ExpandedIERC20ExclusiveMinter(_tokenAddress);
+
+        // TODO: Check that tokenCurrency has an Exclusive, not Shared, Minter role.
+        require(tokenCurrency.isMinterExclusive(), "Token minter role not exclusive");
```

Comment:
This line is added since this contract no longer creates the SyntheticToken. It enforces that the inputted SyntheticToken only has one minter, which has to be this contract if it is to create any tokens.